### DISCUSSION TITLE
How can I install on arm64?

### DISCUSSION THREAD

**Discussion by tobip** [starship/starship#2337] [Q&A] [answered]
I am running Manjaro arm64 on a PineBook Pro.
First I tried to install through the AUR packages, giving me the `ERROR: starship is not available for the 'aarch64' architecture.`.
When I download and run the arm64 binary, it just ends with `Illegal instruction (core dumped)`.
I don't know how I can run starship on my system.

**Comment by davidkna** [comment, marked as answer]
To make sure: Did you download `starship-aarch64-unknown-linux-musl.tar.gz` from the release section on GitHub and it failed to run on your system with a warning about Illegal instructions?

As for the AUR issue most AUR installers have the option of ignoring the specified arch target and installing a package anyway. 
@chipbuster may also be able to add your target to the PKGBUILD or change it to `any` to allow you to install it.

**Reply by tobip** [reply]
Whoops my bad, I downloaded the arm version haha, of course, aarch64 works on my system! Thanks a lot :-)
